src/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.cs

```diff
@@ -60,13 +61,235 @@ public partial class AppSettingsReader
         public AppSettingsReader() { }
         public object GetValue(string key, System.Type type) { throw null; }
     }
+    public sealed partial class AppSettingsSection : System.Configuration.ConfigurationSection
+    {
+        public AppSettingsSection() { }
+        [System.Configuration.ConfigurationPropertyAttribute("file", DefaultValue = "")]
+        public string File { get { throw null; } set { } }
+        protected internal override System.Configuration.ConfigurationPropertyCollection Properties { get { throw null; } }
+        [System.Configuration.ConfigurationPropertyAttribute("", IsDefaultCollection = true)]
+        public System.Configuration.KeyValueConfigurationCollection Settings { get { throw null; } }
+        protected internal override void DeserializeElement(System.Xml.XmlReader reader, bool serializeCollectionKey) { }
+        protected internal override object GetRuntimeObject() { throw null; }
+        protected internal override void Reset(System.Configuration.ConfigurationElement parentSection) { }
+    }
+    public sealed partial class CallbackValidator : System.Configuration.ConfigurationValidatorBase
+    {
+        public CallbackValidator(System.Type type, System.Configuration.ValidatorCallback callback) { }
+        public override bool CanValidate(System.Type type) { throw null; }
+        public override void Validate(object value) { }
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property)]
+    public sealed partial class CallbackValidatorAttribute : System.Configuration.ConfigurationValidatorAttribute
+    {
+        public CallbackValidatorAttribute() { }
+        public string CallbackMethodName { get { throw null; } set { } }
+        public System.Type Type { get { throw null; } set { } }
+        public override System.Configuration.ConfigurationValidatorBase ValidatorInstance { get { throw null; } }
+    }
     public sealed partial class ClientSettingsSection : System.Configuration.ConfigurationSection
     {
         public ClientSettingsSection() { }
-        protected override System.Configuration.ConfigurationPropertyCollection Properties { get { throw null; } }
+        protected internal override System.Configuration.ConfigurationPropertyCollection Properties { get { throw null; } }
```

Comment:
Should we explicitly have protected internal in the ref? Is it needed? 

cc @maryamariyan, @safern, @stephentoub

Reply:
Technically it's different, but since we don't have an InternalsVisibleTo it's not relevant... if the tool wrote `protected internal` it's fine.